tests/sentry/web/frontend/test_demo_start.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from sentry.utils.compat import mock
+
+from django.test.utils import override_settings
+from django.template.defaultfilters import slugify
+from exam import fixture
+
+from sentry.testutils import TestCase
+from sentry.models import (
+    User,
+    Organization,
+    OrganizationMember,
+    Project,
+    ProjectKey,
+)
+from sentry.utils.email import create_fake_email
+
+org_owner_email = "elon@tesla.com"
```

Comment:
Uh, no please. I don't know if there's even legality issues with doing this, but stick to `@example.com` or `@sentry.io` emails for tests.

Reply:
lol fair enough, updated another test I had where I used this